ee/api/explicit_team_member.py

```diff
@@ -39,7 +39,7 @@ class TeamMemberObjectPermissions(BasePermission):
         and at least admin effective project access level for write/delete.
     """
 
-    message = "You don't have sufficient permissions in this project."
+    message = "Your project access level is too low for this action."
```

Comment:
Personally I think the previous message is better from a UX standpoint. The concept of permission levels being integers and "higher" / "lower" seems more complex to grasp for users.